src/module_2/download_data_from_s3.py

```diff
@@ -0,0 +1,100 @@
+import boto3
+from dotenv import load_dotenv
+import os
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+
+def load_variables():
+    load_dotenv()
+
+    access_key = os.getenv("ACCESS_KEY_ID")
+    secret_key = os.getenv("SECRET_ACCESS_KEY")
+    bucket_name = os.getenv("BUCKET_NAME")
+    s3_directory = "groceries/box_builder_dataset/"
+    local_directory = "/home/manucorujo/zrive-ds-data/"
```

Comment:
You can use relative paths to avoid adding personal information about your filesystem